pkg/kube/inject/webhook.go

```diff
@@ -611,6 +611,7 @@ func (wh *Webhook) inject(ar *kube.AdmissionReview, path string) *kube.Admission
 	log.Debugf("Object: %v", string(req.Object.Raw))
 	log.Debugf("OldObject: %v", string(req.OldObject.Raw))
 
+	wh.mu.RLock()
```

Comment:
unlock in L618

Reply:
good catch. we should refactor this code a bit in the future so we can access everything from one function